lib/rufo/formatter/settings.rb

```diff
@@ -1,93 +1,29 @@
 class Rufo::Formatter
-  def init_settings(options)
-    spaces_around_binary options.fetch(:spaces_around_binary, :dynamic)
-    parens_in_def options.fetch(:parens_in_def, :yes)
-    double_newline_inside_type options.fetch(:double_newline_inside_type, :dynamic)
-    align_case_when options.fetch(:align_case_when, false)
-    align_chained_calls options.fetch(:align_chained_calls, false)
-    trailing_commas options.fetch(:trailing_commas, :always)
-  end
-
-  def spaces_around_binary(value)
-    @spaces_around_binary = one_dynamic("spaces_around_binary", value)
-  end
+  OPTIONS = {
+    spaces_around_binary: [:dynamic, :one],
+    parens_in_def: [:yes, :dynamic],
+    double_newline_inside_type: [:dynamic, :no],
+    align_case_when: [false, true],
+    align_chained_calls: [false, true],
+    trailing_commas: [:always, :never],
+  }
 
-  def parens_in_def(value)
-    @parens_in_def = yes_dynamic("parens_in_def", value)
-  end
-
-  def double_newline_inside_type(value)
-    @double_newline_inside_type = no_dynamic("double_newline_inside_type", value)
-  end
+  attr_accessor *OPTIONS.keys
```

Comment:
Do we need real setters and getters?

Currently, these are just instance variables.